### ISSUE TITLE
Remove non-inclusive terminology from aws_availability_zones data source attribute names

### ISSUE THREAD

**Issue by breathingdust** [terraform-providers/terraform-provider-aws#13766]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Use the next major release as an opportunity to remove non-inclusive terminology in the provider codebase, documentation, and examples.  We will be removing words whose meanings, origin, or associations are contrary to our [Principles](https://www.hashicorp.com/our-principles/).

### New or Affected Resource(s)

* aws_availability_zones

### Potential Terraform Configuration

 Before:

```hcl
data "aws_availability_zones" "test" {
  blacklisted_zone_ids = []
  blacklisted_names = []
}
```

After:

```hcl
data "aws_availability_zones" "test" {
  skip_zone_ids = []
  skip_names = []
}
```

### References

*  [HashiCorp Principles](https://www.hashicorp.com/our-principles/)

**Comment by bflad** [comment]
First pass: https://github.com/terraform-providers/terraform-provider-aws/pull/13771

**Comment by chrisfowles** [comment]
Ironically `exclude` is probably better and less ambiguous terminology here.

i.e.
```hcl
data "aws_availability_zones" "test" {
  exclude_zone_ids = []
  exclude_names = []
}
```

**Comment by myoung34** [comment]
`skip` has some connotations to it about not being evaluated.
`exclude` makes more sense as a replacment term

**Comment by clort81** [comment]
This is bullshit.  If offends me.

**Comment by chrisfowles** [comment]
@myoung34 - while i agree they're similar, I feel that the intended action inflects different intents on the terms.

> `skip` has some connotations to it about not being evaluated.
> `exclude` makes more sense as a replacement term

in my understanding of common parlance.

`skip` is something that you do over a single item or set of items in a collection that you are iterating through. The connotations for skip for filtering are usually in terms of a number of records.

e.g. i want to read the this file but skip the first 2 lines.

`exclude` makes more sense on a data source as a filter.

e.g. i want to read this file but exclude any lines starting with cat.

**Comment by opusmode** [comment]
Sounds stupidly and useless

**Comment by paultyng** [comment]
@clort81 I edited your comment, the original comment was not in line with our [community guidelines](https://www.hashicorp.com/community-guidelines)

People are free to discuss the technical considerations of this change, but as @breathingdust states in the original post, this is to bring the code in line with our internal standards, not a dicussion of what those standards are.

**Comment by lukashin** [comment]
@paultyng @breathingdust could you please underline which principle word blacklist is opposing and how exactly?

This is a big breaking change and it diverts terraform terminology from one used by AWS (and community in general) leading to confusion into critical infrastructure.
There is no negative connotations, "meanings, origin, or associations" to the word black that I can recall neither in English nor Russian language (native for me). 

Would you share whole list of words that supposed to be excluded from language according to hashicorp and replaced by Newspeak? 

Below I'm leaving a link to master/slave discussion in Redis and specifically comment by Sam Parker D
> I don't need to be coddled or be considered so stupid I can't tell the difference between slavery and a cluster. That's insulting.
http://antirez.com/news/122#comment-4084872912

**Comment by paultyng** [comment]
@lukashin this is a pretty bad faith reading of everything being discussed. The original post which lists the specifics, and the referenced changes are a single data source and NOT upstream AWS API rendering much of your comments here moot.

I've hidden your response as it was also explicitly counter to what I previously said that this issue is not for discussion of the standards.  Please refrain from further similar comments elsewhere.

**Comment by bflad** [comment]
Removal of the deprecated `aws_availabiliity_zones` arguments has been merged and will be released with version 3.0.0 of the Terraform AWS Provider, in about two weeks.
